src/operators/video_effects.ml

```diff
@@ -208,14 +225,14 @@ let () =
 let () =
   Lang.add_operator "video.opacity.blur"
     [
-      "", Lang.source_t kind, None, None
+      "", Lang.source_t return_t, None, None
     ]
     ~return_t
     ~category:`Video
     ~descr:"Blur opacity of video."
     (fun p ->
        let src = Lang.to_source (Lang.assoc "" 1 p) in
-         new effect ~kind Image.Effect.Alpha.blur src)
+         new effect ~kind src Image.YUV420.Effect.Alpha.blur)
 
 let () =
   Lang.add_operator "video.lomo"
```

Comment:
Is that dead code?

Reply:
Yes, they should be restored at some point, but it's not easy on YUV representation (#1066).